core/Nonce.php

```diff
@@ -123,7 +123,7 @@ public static function verifyNonceWithErrorMessage($id, $cnonce, $allowedReferre
             // Allow the instance host by default, if no allowedReferrerHost is specified.
             if (empty($allowedReferrerHost) && !Url::isLocalUrl($referrer)) {
                 return Piwik::translate('Login_InvalidNonceReferrer', array(
-                        '<a target="_blank" rel="noreferrer noopener" href="https://matomo.org/faq/how-to-install/faq_98">',
+                        '<a target="_blank" rel="noreferrer noopener" href="' . Url::addCampaignParametersToMatomoLink('https://matomo.org/faq/how-to-install/faq_98') . '">',
                         '</a>'
```

Comment:
and then we could also do an optional param which would make it return an array of the opening and the closing tags, usable with the destructuring `...` in arrays.
```suggestion
                        ...Url::getExternalMatomoLinkTag('https://matomo.org/faq/how-to-install/faq_98', true),
```

Reply:
Might be too much magic for some people's appetite though 🤷